packages/phasor/src/utils/line.ts

```diff
@@ -0,0 +1,38 @@
+import { clamp } from './math-utils.js';
+import { almostEqual, EPSILON, MACHINE_EPSILON } from './math-utils.js';
+import type { Point } from './point.js';
+
+export class Line {
```

Comment:
Looks better to have `lineIntersects` in `math-utils`  rather than adding a redundant class. Also it's suggested to make it depend on `Vec` instead of `Point`.

Reply:
`Vec.cpr` and `Vec.lrp` are the existing methods to be reused.

Reply:
okay